Libraries/LibWeb/IndexedDB/Internal/Algorithms.h

```diff
@@ -31,5 +31,6 @@ void commit_a_transaction(JS::Realm&, GC::Ref<IDBTransaction>);
 WebIDL::ExceptionOr<JS::Value> clone_in_realm(JS::Realm&, JS::Value, GC::Ref<IDBTransaction>);
 ErrorOr<GC::Ref<Key>> convert_a_value_to_a_multi_entry_key(JS::Realm&, JS::Value);
 WebIDL::ExceptionOr<ErrorOr<JS::Value>> evaluate_key_path_on_a_value(JS::Realm&, JS::Value, KeyPath);
+WebIDL::ExceptionOr<ErrorOr<GC::Ref<Key>>> extract_a_key_from_a_value_using_a_key_path(JS::Realm&, JS::Value, KeyPath, bool = false);
```

Comment:
prefer Yes/No enums to opaque bool params, especially if you don't want to name them :sweat_smile:

Reply:
The spec calls this method passing in a boolean.. Should that do some bool to enum conversion then?